fulp_modules/devils/devil_powers.dm

```diff
@@ -0,0 +1,157 @@
+/// Summon Pitchfork
+/obj/effect/proc_holder/spell/targeted/conjure_item/summon_pitchfork
+	name = "Summon Pitchfork"
+	desc = "A devil's weapon of choice.  Use this to summon/unsummon your pitchfork."
+	invocation_type = "none"
+	include_user = TRUE
+	range = -1
+	clothes_req = FALSE
+	item_type = /obj/item/pitchfork/demonic
+	school = "conjuration"
+	charge_max = 150
+	cooldown_min = 10
+	action_icon = 'fulp_modules/devils/devil_icon/devil_icons.dmi'
+	action_icon_state = "pitchfork"
+	action_background_icon_state = "bg_demon"
+
+/obj/item/pitchfork/demonic
+	name = "demonic pitchfork"
+	desc = "A red pitchfork, it looks like the work of the devil."
+	lefthand_file = 'fulp_modules/devils/devil_icon/devils_lefthand.dmi'
+	righthand_file = 'fulp_modules/devils/devil_icon/devils_righthand.dmi'
+	force = 19
+	throwforce = 24
+	light_system = MOVABLE_LIGHT
+	light_range = 3
+	light_power = 6
+	light_color = COLOR_SOFT_RED
+
+/obj/item/pitchfork/demonic/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_unwielded=19, force_wielded=25)
+
+/obj/item/pitchfork/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] repeatedly stabs [user.p_them()]selves in [user.p_their()] abdomen with [src]! It looks like [user.p_theyre()] trying to send themselves to Hell!</span>")
+	playsound(loc, 'sound/hallucinations/veryfar_noise.ogg', 50, TRUE, -1)
+	sleep(20)
```

Comment:
ewwwwww, a sleep()

Reply:
TG coders when someone goes to sleep rather than have a cooldown signal